jetson/percep/pcl.cpp

```diff
@@ -186,185 +188,79 @@ void PCL::FindInterestPoints(std::vector<pcl::PointIndices> &cluster_indices,
 
 }
 
+/* --- Get Angle Off Center--- */
+//This function finds the angle off center the
+//line that passes through both these points is
+//Direction of 0 is left and 1 is right
+double PCL::getAngleOffCenter(int buffer, int direction, const std::vector<std::vector<int>> &interest_points,
+                    shared_ptr<pcl::visualization::PCLVisualizer> viewer, std::vector<int> &obstacles){
+    double newAngle = 0;
+    //If Center Path is blocked check the left or right path depending on direction parameter
+    while(newAngle > -70 && newAngle < 70) {
```

Comment:
Make 70 a constant at top of file. E.g.:
```
const int MAX_FIELD_OF_VIEW_ANGLE = 70
```

Reply:
Also, add this item to #474's list of items to add to the config file. This list will not be complete but it will be a good idea to add items to that list as you come across them to save yourself work in the future.

Reply:
Added